kubeflow/fairing/preprocessors/function.py

```diff
@@ -22,18 +22,25 @@
 
 
 class FunctionPreProcessor(BasePreProcessor):
-    """
-    FunctionPreProcessor preprocesses a single function.
-    It sets as the command a function_shim that calls the function directly.
+    """FunctionPreProcessor preprocesses a single function.
+       It sets as the command a function_shim that calls the function directly.
+    :param BasePreProcessor: a context that gets sent to the builder for the docker build and \
```

Comment:
I think no need "\" here, @xauthulei could you please confirm?

Reply:
I have deleted it, Thanks for your great efforts here. @jinchihe